tests/integration/build_invoke/test_build_invoke_dotnetcore2_1.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from tests.integration.base import Base
+
+# sam build is likely to fail in the first try because dotnet.lambda cannot be used immediately after installation
```

Comment:
This is an interesting gotcha. why is that the case? is it not part of the PATH?

Reply:
Resolved in https://github.com/aws/aws-sam-cli-app-templates/pull/66#discussion_r536433899